### ISSUE TITLE
Changing <del/> display conventions

### ISSUE THREAD

**Issue by ajaniak** [erc-dharma/project-documentation#140]
Dear all, 

Arlo suggests to change DHARMA display convention for the element `<del>` from {{ }}, which is currently applied, to ⟦ ⟧. Can we have your opinion on the matter. 

Thanks

**Comment by AnneSchmiedchen** [comment]
What is the reason for that suggeted change?

**Comment by danbalogh** [comment]
I'm perfectly OK with ⟦ ⟧ for `<del>`; this in fact may be exactly what I suggested at the very beginning. It's certainly in accordance with Leiden and all the conventions of classical epigraphers shown in the EpiDoc cheatsheet, and I think it is a good idea to harmonise our conventions with those where feasible.
I think the original rationale behind choosing {{ }} was that we use { } for editorial deletion (same in Leiden) and ⟨ ⟩ for editorial addition (same in Leiden), and we decided to use the same signs doubled (i.e. {{ }} and ⟨⟨ ⟩⟩ ) for the display of scribal deletion and addition. But if nobody insists on that kind of harmony, then let's go for ⟦ ⟧ in scribal deletion.
Going out on a limb, the Leiden display of scribal addition is something else again, since they use three different methods for indicating A, text written over an erasure; B, text added above the line and C, text added below the line. The first of these is done with《 》in the Krummrey-Panciera transliteration, but not in any other system shown on the EpiDoc cheatsheet. The characters《 》appear to be supported only in far-eastern fonts (block "CJK symbols and punctuation"), so I'm not suggesting we adopt it. Nor am I suggesting that we adopt a different display for addition above and below, since we have several additional values permitted for the `@place` of `<add>` and I see no point in differentiating some of the permitted values in display while keeping others the same. Better to indicate `@place` in a tooltip.

So, the bottom line: as far as I am concerned, I have a slight preference for changing to ⟦ ⟧  from {{ }}, but I'm also OK with keeping the present system. And I suggest that even if we make this change, we should make no other changes to related display cases.
